lavamoat/browserify/beta/policy.json

```diff
@@ -3728,6 +3728,12 @@
         "@babel/runtime": true
       }
     },
+    "dompurify": {
+      "globals": {
+        "console.warn": true,
```

Comment:
Console warn can be found in the dompurify source code.
https://github.com/cure53/DOMPurify/blob/main/src/purify.ts#L98